### ISSUE TITLE
Install pyDive - mpirun

### ISSUE THREAD

**Issue by ax3l** [ComputationalRadiationPhysics/pyDive#15]
[Installing](https://computationalradiationphysics.github.io/pyDive/start.html?highlight=install) `pyDive` fails on hypnos if not run via `mpirun` (_during_ install).

Example:

``` bash
mpirun -n 1 easy_install --prefix=$HOME/ipython/local pyDive
#or
mpirun -n 1 python setup.py install --prefix=$HOME/ipython/local
```

Should we update the documentation to reflect that?

**Comment by ax3l** [comment]
additionally, since the automated install script is not in the `PYTHONPATH` for packages (?) ~~I had to add a softlink to the parent directory in `arrays/` as this: `pyDive -> ../`. -> update: does not solve the problem.~~

the import of pyDive dies here: https://github.com/ComputationalRadiationPhysics/pyDive/blob/7d74819fbe6e5b0f70fcc1ef9f32a2c87870651f/pyDive/arrays/ndarray.py#L24

adding [this code](http://stackoverflow.com/a/11158224) in the according files (such as [h5_ndarray.py](https://github.com/ComputationalRadiationPhysics/pyDive/blob/7d74819fbe6e5b0f70fcc1ef9f32a2c87870651f/pyDive/arrays/h5_ndarray.py#L24)) helped with python 2.1

**Comment by heikoburau** [comment]
Before you install any packages for pyDive or pyDive itself you need to install and run `python virtual environment` first. Then you install each package via pip or by `python setup.py install` wihtout `--prefix`.
Running the virtual environment is the python analogon for e.g. `source picongpu.profile`.

**Comment by ax3l** [comment]
oh, [that virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/) ? I did not know that, it's a real cool feature!
If it's required can you pls add it to the docs? :) I just read [the quickstart document](https://computationalradiationphysics.github.io/pyDive/start.html#quickstart) to install pyDive - did I miss it?

**Comment by heikoburau** [comment]
Yes that's the right link, I should have mentioned it before.
Actually it's not _required_, but it helps a lots to have a clean python environment, with just the packages you need in the right place and in the right version. But we can add a little hint window in the quickstart. I guess the trouble above comes from a dirty environment with packages floating around anywhere in your linux system.

**Comment by ax3l** [comment]
that's unfortunately quite likely on clusters :)

**Comment by ax3l** [comment]
problem now solved with 76634a1f3028b7cdbed2e8dfb7fdc118ed3921ff (pip install caused `File "pyDive/arrays/h5_ndarray.py", line 29, in <module>      import pyDive.arrays.local.h5_ndarray  ImportError: No module named local.h5_ndarray`)

and with b7177b111742dd81c07feea32937a7d260aaff03 the automated install of requirements during pip installs should also be fixed
